sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/CommunicationTokenRefreshOptions.java

```diff
@@ -59,9 +59,9 @@ public boolean isRefreshProactively() {
     }
 
     /**
-     * @return the serialized JWT token
+     * @return the initial serialized JWT token
```

Comment:
Let's remove "serialized JWT". The format is an implementation detail. Let's just say "the initial token."